### DISCUSSION TITLE
Review function for the app.

### DISCUSSION THREAD

**Discussion by Draper007** [FilledStacks/boxtout#29] [Ideas]
Firstly thanks for the videos, Great help, videos have filled in huge gaps in workflow for me. 
However, I think a customer review function within the app would be great. Having worked in the food business for over 20 years I can tell you that reviews are the only thing that helps drive the team. you always need customer feedback,, is a big KPI for the food business. 
However online review platforms are abundant, I would suggest turning the tables and having a private, not public review portal. This could work in two ways, customers will have a portal to complain without shooting off public reviews, so more people might use the service. Each review is then sent to every employee of the business via SMS or email, for them to know the team's performance.   these reviews are used to improve services and offerings and not used for sale and marketing purposes. 

public or private a feedback portal is a must  

once again thanks for starting the project,

**Comment by FilledStacks** [comment]
@Draper007 thanks for leaving a detailed comment. I appreciate it. 

That does sound like a cool system to build, it sounds like something like that might exist yet, if not it's a good opportunity to build some software you can setup for internal reviews from a mobile app or website. 

This will most likely be far after having the food delivery functionality built. Until we have the functionality to place an order, take payment, get that delivered to a location we can't really focus on review systems 😄 but it's something that will stay here and we can definitely add it in and maybe even get some software out of it for FilledStacks to use for other clients.
